mods/cnc/maps/nod06b/map.yaml

```diff
@@ -0,0 +1,661 @@
+		Location: 54,24
+		Owner: GDI
+	Actor29: sbag
+		Location: 54,23
+		Owner: GDI
+	Actor30: sbag
+		Location: 54,22
+		Owner: GDI
+	Actor31: sbag
+		Location: 54,21
+		Owner: GDI
+	Actor32: sbag
+		Location: 40,21
+		Owner: GDI
+	Actor33: sbag
+		Location: 54,20
+		Owner: GDI
+	Actor34: sbag
+		Location: 40,20
+		Owner: GDI
+	Actor35: sbag
+		Location: 54,19
+		Owner: GDI
+	Actor36: sbag
+		Location: 40,19
+		Owner: GDI
+	Actor37: sbag
+		Location: 54,18
+		Owner: GDI
+	Actor38: sbag
+		Location: 40,18
+		Owner: GDI
+	Actor39: sbag
+		Location: 54,17
+		Owner: GDI
+	Actor40: sbag
+		Location: 53,17
+		Owner: GDI
+	Actor41: sbag
+		Location: 52,17
+		Owner: GDI
+	Actor42: sbag
+		Location: 51,17
+		Owner: GDI
+	Actor43: sbag
+		Location: 50,17
+		Owner: GDI
+	Actor44: sbag
+		Location: 49,17
+		Owner: GDI
+	Actor45: sbag
+		Location: 48,17
+		Owner: GDI
+	Actor46: sbag
+		Location: 47,17
+		Owner: GDI
+	Actor47: sbag
+		Location: 46,17
+		Owner: GDI
+	Actor48: sbag
+		Location: 45,17
+		Owner: GDI
+	Actor49: sbag
+		Location: 44,17
+		Owner: GDI
+	Actor50: sbag
+		Location: 43,17
+		Owner: GDI
+	Actor51: sbag
+		Location: 42,17
+		Owner: GDI
+	Actor52: sbag
+		Location: 41,17
+		Owner: GDI
+	Actor53: sbag
+		Location: 40,17
+		Owner: GDI
+	Actor54: t18
+		Location: 28,31
+		Owner: Neutral
+	Actor55: t08
+		Location: 25,26
+		Owner: Neutral
+	Actor56: t08
+		Location: 27,21
+		Owner: Neutral
+	Actor57: t18
+		Location: 25,20
+		Owner: Neutral
+	Actor58: t18
+		Location: 33,45
+		Owner: Neutral
+	Actor59: t18
+		Location: 23,48
+		Owner: Neutral
+	Actor60: t08
+		Location: 52,38
+		Owner: Neutral
+	Actor61: t08
+		Location: 55,47
+		Owner: Neutral
+	Actor62: rock6
+		Location: 24,55
+		Owner: Neutral
+	Actor63: t08
+		Location: 32,58
+		Owner: Neutral
+	Actor64: t08
+		Location: 35,56
+		Owner: Neutral
+	Actor65: t08
+		Location: 30,50
+		Owner: Neutral
+	Actor66: t08
+		Location: 35,52
+		Owner: Neutral
+	Actor67: t08
+		Location: 37,56
+		Owner: Neutral
+	Actor68: t08
+		Location: 16,53
+		Owner: Neutral
+	Actor69: t08
+		Location: 25,61
+		Owner: Neutral
+	Actor70: t08
+		Location: 17,41
+		Owner: Neutral
+	Actor71: t08
+		Location: 49,34
+		Owner: Neutral
+	Actor72: t08
+		Location: 27,48
+		Owner: Neutral
+	Actor73: t08
+		Location: 34,36
+		Owner: Neutral
+	Actor74: t08
+		Location: 25,52
+		Owner: Neutral
+	Actor75: t08
+		Location: 43,30
+		Owner: Neutral
+	Actor76: rock1
+		Location: 45,29
+		Owner: Neutral
+	Actor77: t18
+		Location: 48,44
+		Owner: Neutral
+	Actor78: t18
+		Location: 58,45
+		Owner: Neutral
+	Actor79: t18
+		Location: 35,38
+		Owner: Neutral
+	Actor80: t18
+		Location: 50,55
+		Owner: Neutral
+	Actor81: t18
+		Location: 23,15
+		Owner: Neutral
+	Actor82: t18
+		Location: 15,13
+		Owner: Neutral
+	Actor83: t08
+		Location: 39,14
+		Owner: Neutral
+	Actor85: nuke
+		Location: 44,18
+		Owner: GDI
+	Actor86: v35
+		Location: 36,57
+		Owner: Neutral
+	Actor87: v34
+		Location: 34,56
+		Owner: Neutral
+	Actor88: v33
```

Comment:
Nod units don't automatically target the civilian stuff, which normally happens even when it's not an explicit mission objective.
It would be nice if you could add a non-combatant player as enemy for Nod and assign all the civilian things to it.